# we must set am__api_version to the installed version
# to avoid abort with error: 
# LibRaw-0.20.2/missing: line 81: aclocal-1.15: command not found
# and we must run aclocal before ./configure
# 
--- configure.orig	2020-09-29 08:18:53.000000000 +0000
+++ configure	2022-09-02 11:15:35.684085123 +0000
@@ -2395,7 +2395,7 @@
 ac_compiler_gnu=$ac_cv_c_compiler_gnu
 
 
-am__api_version='1.15'
+am__api_version='1.16'
 
 ac_aux_dir=
 for ac_dir in "$srcdir" "$srcdir/.." "$srcdir/../.."; do
